demo/public/lib/lazy.js

```diff
@@ -3,14 +3,19 @@ import { useState, useRef } from 'preact/hooks';
 
 export default function lazy(load) {
 	let p, c;
-	return props => {
+	function inner(props) {
 		if (!p) p = load().then(m => ((c = (m && m.default) || m), 1));
 		const [, update] = useState(0);
 		const r = useRef(c);
 		if (!r.current) r.current = p.then(update);
 		if (c === undefined) throw p;
+		inner._r = true;
```

Comment:
Let's give this a full name rather than `_r` (`_resolved`) and work with mangle (in the future) to better this.

Reply:
Oh, thank you @JoviDeCroock! Great call. Will change it.